src/style.css

```
@@ -0,0 +1,75 @@
+* {
+  margin: 0;
+}
+
+.body {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+[class^="block"] {
```

```suggestion
.block {
```
Этого будет достаточно, для order/bg color используй nth-child